### PR TITLE
Get started: describe how to use CLI

### LINKED ISSUE #1597: Add get started for CLI (non-Webstorm)
Tell folks how to install/use:

- [ ] stagehand
- [ ] webdev

Related to https://github.com/dart-lang/site-webdev/issues/1467

### PR DESCRIPTION
**Pull request by chalin**
I avoided placing WS-vs-CLI instructions under tabs or subheadings by using icons as markers. This is a useful eye-catcher because the WS-vs-CLI instructions are spread across a few steps.

Fixes #1597

Staged at https://angulardart-org-dev2.firebaseapp.com/guides/get-started

cc @kevmoo 

Here is what it looks like for the updated Step 3:

screen shot 2018-05-23 at 13 31 43

### CODE REVIEW

**Comment by kevmoo** [comment]
Awesome!

File: src/_guides/get-started.md [hunk 1]
```diff
@@ -108,18 +108,31 @@ You'll need it in step 4.
...
-Although using an IDE is optional, we highly recommend WebStorm. WebStorm comes
+<i class="material-icons">web</i>
+Although using an IDE is optional, we highly recommend that you
+[download and install WebStorm](/tools/webstorm). WebStorm comes
 with Dart support, making it easy to write correct Dart code and to run it in a
 browser.
 
-[Download and install WebStorm](/tools/webstorm).
+<i class="fas fa-terminal dark"></i>
+If you'd rather work from the command line, install the following tools:
```
Comment (kwalrath) [inline src/_guides/get-started.md]: Let's name and link to the tools here, for clarity, and then we can remove them from the "More information" list.

`the following tools:` -> `[webdev](...) and [stagehand:](...)`

File: src/_guides/get-started.md [hunk 2]
```diff
@@ -108,18 +108,31 @@ You'll need it in step 4.
...
 
-More information: [Dart tools for the web](/tools)
+```terminal
+$ pub global activate webdev
+$ pub global activate stagehand
+```
+
+More information:
+
+- [Dart tools for the web](/tools)
```
Comment (kwalrath) [inline src/_guides/get-started.md]: Revert the "More information" part of this step.

File: src/_guides/get-started.md [hunk 3]
```diff
@@ -135,10 +148,21 @@ Here's how to use WebStorm to create a web app that uses AngularDart:
 
 More information: [Setup for AngularDart development](/angular/guide/setup)
```
Comment (kwalrath) [inline src/_guides/get-started.md]: Move "More information" to the end of this step.

File: src/_guides/get-started.md [hunk 4]
```diff
@@ -135,10 +148,21 @@ Here's how to use WebStorm to create a web app that uses AngularDart:
...
 
+<i class="fas fa-terminal dark"></i>
+Create the app from the command line using these commands:
```
Comment (kwalrath) [inline src/_guides/get-started.md]: -> To create the app from the command line, use these commands:

File: src/_guides/get-started.md [hunk 5]
```diff
@@ -152,6 +176,17 @@ You should see a simple todo list manager. Try it out!
...
+<i class="fas fa-terminal dark"></i>
+To run the app from the command line, use [webdev][] to build and serve the app:
+
+```terminal
+$ webdev serve
+```
+
+Then, to view your app, visit [localhost:8080](localhost:8080).
+Webdev is slowest when it builds and serves your app for the first time.
+After that, assets are cached on disk and incremental builds are faster.
```
Comment (kwalrath) [inline src/_guides/get-started.md]: faster -> much faster
?
("faster" sounds like it might not be fast.)

**Comment by kwalrath** [comment]
I like how you changed the terminal prompt. I thought about that but then forgot to mention it.

**Comment by chalin** [comment]
👍 ... it seemed reasonable to match the icon more closely.

**Pull request merged**